requirements_dev.in

```diff
@@ -0,0 +1,3 @@
+-r requirements.in
```

Comment:
Normally we  use .txt not .in here. That way we can be sure that the versions of libs the app needs are the same as are specified in requirements.txt - if .in is used there is a risk they could get out of sync if pip-compile is run on different files at different times.

Can you change that and re-run so requirements_dev.txt updates?

Reply:
Ah that makes sense, will update now.

Reply:
Updated